gp-templates/profile-view.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
I flip flopped a few times on it but it seemed better to keep in in the core as it's really a "read only" version of the edit file.

Since custom theme's might make changes to the edit file I thought it best that they have the opportunity to change the view file at the same time to retain a consistent look/feel.